src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -222,11 +222,24 @@ describe('Reimbursement Requests', () => {
           [],
           superman
         )
-      ).rejects.toThrow(
-        new AccessDeniedException(
-          'You do not have access to delete this reimbursement request, only the creator can edit a reimbursement request'
+      ).rejects.toThrow(new AccessDeniedException('You do not have access to edit this reimbursement request'));
```

Comment:
Let's update this error msg after we resolve the error msg from the service function.